ksqldb-execution/src/test/java/io/confluent/ksql/execution/codegen/helpers/CastEvaluatorTest.java

```diff
@@ -729,6 +729,7 @@ static Object instanceFor(final SqlType type, final SqlType to) {
           if (to.baseType() == TIMESTAMP) {
             return "2020-05-26T07:59:58.000";
           }
+          // Intentional fall through
```

Comment:
Do we need this?

Reply:
Not strictly. Saw it in another test with the same warning. -- Guess it's good to have for clarification.